pages/common/nu.md

```diff
@@ -0,0 +1,24 @@
+# nu
+
+> Nushell, a "new type of shell": takes a modern, structured approach to your commandline.
```

Comment:
```suggestion
> Nushell - a modern, structured approach to your commandline.
```
This sounds a bit more tldr-conform, what do you think?

Reply:
I prefer my take that mixes the official moto (`a "new type of shell"`, that explain the name) with my own explanation (`structured approach to your commandline`). 😇

Reply:
Yeah, it does, but I'm not sure about the hypen used there 🤔
Perhaps this?
```suggestion
> Nushell (new shell) takes a modern, structured approach to your command-line.
```

Reply:
OK 👍. I did a mix of your comments and mine in commit e7b66f4.